.github/workflows/pull_request.yml

```diff
@@ -1,6 +1,19 @@
 name: Build Chain
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'LICENSE*'
+      - 'CODEOWNERS'
+      - '.gitignore'
+      - '.gitattributes'
+      - '.travis.yml'
+      - '*.md'
+      - '*.adoc'
+      - '*.txt'
+      - 'runOnOpenShift.sh'
+      - 'runLocally.sh'
```

Comment:
`.ci/**` ?

Reply:
@radtriste because we want to trigger `.ci` changes for an action using build-chain tool and `.github/workflows/pull_request.yml` does it

Reply:
but taking changes from droolsjbpm-build-bootstrap... so I will add it

Reply:
```suggestion
      - 'runLocally.sh'
      - '.ci/**'
```